r2angr/angr

```diff
@@ -3,11 +3,78 @@ CWD=$(dirname $0)
 OLD=$PWD
 cd $CWD
 if [ ! -d venv ]; then
-    python -m venv venv
+    if ! command -v termux-setup-storage; then
+        python3 -m venv venv
+    else
+        echo -e "\033[1;33mTermux environment detected. Installing necessary packages...\033[0m"
+
+        pkg update && pkg upgrade -y
+        pkg install -y cmake ninja clang python-numpy unicorn # uncorn Required for unicorn during angr installation
+        python3 -m venv --system-site-packages venv           # For packages like numpy
+    fi
 fi
 . venv/bin/activate
 if [ ! -x venv/bin/angr ]; then
-    pip install angr
+    if command -v termux-setup-storage; then
+        pip install setuptools
+        pip install unicorn==2.0.1.post1
```

Comment:
Maybe we could remove this also but idk why sometimes with angr installation it doesn't completes perfectly.